nss-day-cohort-44/rare-indomitable-icicles
# Rare mm post nav link and register fix

# ISSUE TICKET 
## Description
I fixed on register page for the second verify password field to be the bio field. I also re-inserted the post tab and all posts show up. 

## Type of change
- [ x ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions
`git fetch origin `
`git checkout rare-mm-css` 
`npm start`

User Instructions: 
Test that the register page has one verify password field and one bio field. Test that there is a Post tab in the nav bar and it shows all posts. 

# Checklist:
